src/Tutorial/PatientBrowser/index.html

```diff
@@ -0,0 +1,288 @@
+        myMSALObj.acquireTokenSilent(requestObj).then(function(tokenResponse) {
+            for (var i=0; i<Object.keys(data.entry).length; i++) {
+                resource = JSON.stringify(data.entry[i].resource);
+                callFHIRServer(FHIRConfig.FHIRendpoint + '/' + data.entry[i].resource.resourceType, 'POST', resource, tokenResponse.accessToken, FHIRCallback)
+            }   
+        });
+    }
+
+    function showWelcomeMessage() {
+        var divWelcome = document.getElementById('WelcomeMessage');
+        divWelcome.innerHTML = "Welcome " + myMSALObj.getAccount().userName + " to FHIR Patient Browsing App";
+        var loginbutton = document.getElementById('SignIn');
+        loginbutton.innerHTML = 'Sign Out';
+        loginbutton.setAttribute('onclick', 'signOut()')
+    }
+    function authRedirectCallBack(error, response) {
+        if (error) {
+            console.log(error);
+        } else {
+            if (response.tokenType === "access_token") {
+                callFHIRServer(FHIRConfig.FHIRendpoint + '/Patient','GET', null, response.accessToken, FHIRCallback);
+            } else {
+                console.log("token type is:" + response.tokenType);
+            }
+        }
+    }
+
+    function json2table_helper(json, i) {
+            var table = document.getElementById("myTable");
+            var row = table.insertRow(-1);
+            // var counter = row.insertCell(0);
+            var lastName = row.insertCell(0);
+            var firstName = row.insertCell(1);
+            var sex = row.insertCell(2);
+            var birthDate = row.insertCell(3);
+            var cellInstruction = row.insertCell(-1);
+            var button = document.createElement('button');
+            var span = document.createElement('span');
+            var updatedNum;
+            if (chooseFileButtonClicked === true) {
+                updatedNum = startidx;
+            }
+            else {
+                updatedNum = i+startidx;
+            }
+            span.setAttribute('id', 'Span_Patient_'+updatedNum);
+            span.innerHTML = " View";
+            button.appendChild(span);
+            cellInstruction.appendChild(button);
+            button.setAttribute('id', 'View_Patient_'+updatedNum);
+            // counter.innerHTML = i+1;
+
+            
+            lastName.innerHTML = JSON.stringify(json.entry[i].resource.name[0].family).replace(/['"]+/g, '');
+            firstName.innerHTML = JSON.stringify(json.entry[i].resource.name[0].given[0]).replace(/['"]+/g, '');
+            sex.innerHTML = JSON.stringify(json.entry[i].resource.gender).replace(/['"]+/g, '');
+            birthDate.innerHTML = JSON.stringify(json.entry[i].resource.birthDate).replace(/['"]+/g, '');
+
+            var eachentry = document.createElement('p');
+            eachentry.setAttribute('id', 'resource bundle '+updatedNum);
+            document.getElementById('json').appendChild(eachentry);
+    }
+
+    function json2table(json) {
```

Comment:
What happens to continuation tokens here, i.e. in the case where there ismore than one patient?